apps/prairielearn/src/pages/jobSequence/jobSequence.html.ts

```diff
@@ -13,31 +12,28 @@ export function JobSequence({
   resLocals: Record<string, any>;
   job_sequence: JobSequenceWithTokens;
 }) {
-  return html`
-    <!doctype html>
-    <html lang="en">
-      <head>
-        ${HeadContents({
-          resLocals,
-          pageTitle: `${job_sequence.description} #${job_sequence.number}`,
-        })}
-        ${compiledScriptTag('jobSequenceClient.ts')}
-      </head>
-      <body>
-        ${Navbar({ resLocals })}
-        <main id="content" class="container-fluid">
-          <h1 class="sr-only">Job Sequence</h1>
-          <div class="row">
-            <div class="col-12">
-              <a class="btn btn-primary mb-4" href="javascript:history.back();">
-                <i class="fa fa-arrow-left" aria-hidden="true"></i>
-                Back to previous page
-              </a>
-            </div>
-          </div>
-          ${JobSequenceResults({ course: resLocals.course, jobSequence: job_sequence })}
-        </main>
-      </body>
-    </html>
-  `.toString();
+  return PageLayout({
+    resLocals,
+    pageTitle: `${job_sequence.description} #${job_sequence.number}`,
+    navContext: {
+      type: 'plain',
+      page: 'admin',
```

Comment:
Some of this is incorrect. Note that this page is mounted in multiple locations in `src/server.ts`, and we need to show different navbars / tab bars depending on where it's mounted. I think the easiest fix would be to pull these values from `res.locals`.

To reproduce, click on a course from the homepage, click "Sync", then "Show server git status". Here's what the navbar looks like on your branch (this is incorrect):

<img width="807" alt="Screenshot 2025-01-24 at 07 34 05" src="https://github.com/user-attachments/assets/9d4a5b3f-3250-4f38-8860-254044a46506" />

And here's what it looks like on master (this is correct):

<img width="792" alt="Screenshot 2025-01-24 at 07 34 25" src="https://github.com/user-attachments/assets/31af7dbc-a478-400c-9bca-74ecac26f9b6" />